atlassian/rest_client.py

```diff
@@ -236,17 +279,37 @@ def request(
             data=data if data else json_dump,
         )
         headers = headers or self.default_headers
-        response = self._session.request(
-            method=method,
-            url=url,
-            headers=headers,
-            data=data,
-            json=json,
-            timeout=self.timeout,
-            verify=self.verify_ssl,
-            files=files,
-            proxies=self.proxies,
-        )
+
+        backoff = 1
+        retries = 0
+        responseloop = True
+        while responseloop:
```

Comment:
Make a endless loop and use break instead of mixing both.
```suggestion
        while True:
```